style.css

```diff
@@ -307,7 +310,6 @@ p {
 .project-description {
   text-align: center;
   font-weight: 500;
```

Comment:
- [ ] Kindly remove the extra text `This is my project description, using the P Tag.` from all your project cards as it is not present in the original design 👍 
- [ ] (OPTIONAL) It would be great to use the correct font color for your project cards title 💯 
- [ ] (OPTIONAL)  Kindly make sure to display your cards in 2 rows of 3 columns in the desktop version. Right now as the desktop screen size increases, the number of columns in your rows varies. It would be great to rectify this issue.
- Your design:
![Annotation 2022-03-26 073019](https://user-images.githubusercontent.com/85113567/160228009-59b308ef-1fd9-404b-804f-8f41e6446961.jpg)
- Figma design:
![Annotation 2022-03-26 073038](https://user-images.githubusercontent.com/85113567/160228016-6372d857-ae74-4560-8012-6936a0e2faab.jpg)

Reply:
Done, Thanks for the observations.
for the third optional change sir, On my screen, it's displayed correctly in 2 rows of 3 columns with all different sizes. Maybe your screen is larger than mine.